<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlguseValikClass</class>
 <widget class="QDialog" name="AlguseValikClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protokollitaja</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Valige fail. Uue loomiseks valige uus. Mõne teise avamiseks valige Lehitse...</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Võistluse nimi:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="voistluseNimi">
       <property name="toolTip">
        <string>Praegu valitava võistluse nimi</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Aeg ja koht:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="aegKohtEdit">
       <property name="toolTip">
        <string>Toimumise aeg ja koht</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Faili nimi:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="failiNimi">
       <property name="toolTip">
        <string>Võistluse faili nimi</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="avaNupp">
       <property name="toolTip">
        <string>Siia vajutades saate otsida teise faili, mida avada</string>
       </property>
       <property name="text">
        <string>Lehitse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="uusNupp">
       <property name="toolTip">
        <string>Siia vajutades saate luua uue võistluse ja faili</string>
       </property>
       <property name="text">
        <string>Uus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edasiNupp">
       <property name="toolTip">
        <string>Selle võistluse ja faili avamiseks vajutage siia</string>
       </property>
       <property name="text">
        <string>Edasi</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="valjuNupp">
       <property name="text">
        <string>Välju</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>valjuNupp</sender>
   <signal>clicked()</signal>
   <receiver>AlguseValikClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
